client/extensions/woocommerce/app/settings/payments/payment-method-edit.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import FormPasswordInput from 'components/forms/form-password-input';
+import FormTextInput from 'components/forms/form-text-input';
+import FormSelect from 'components/forms/form-select';
+import ListItem from 'woocommerce/components/list/list-item';
+import PaymentMethodEditFormToggle from './payment-method-edit-form-toggle';
+
+class PaymentMethodEdit extends Component {
+
+	static propTypes = {
+		method: PropTypes.object,
```

Comment:
Please consider enforcing the shape of the parts of the method prop that you depend on (like that it has a settings property that is an array) and please consider renaming this prop paymentMethod to avoid potential confusion for us old farts that think method === function :)